Packs/GitHubITAdmin/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.yml

```diff
@@ -0,0 +1,268 @@
+      description: Command context path
+      type: unknown
+    - contextPath: GetUser.active
+      description: Gives the active status of user. Can be true of false.
+      type: boolean
+    - contextPath: GetUser.brand
+      description: Name of the Integration
+      type: string
+    - contextPath: GetUser.details
+      description: Gives the user Profile information if the API is success else error
+        message
+      type: string
+    - contextPath: GetUser.email
+      description: Value of email ID passed as argument
+      type: string
+    - contextPath: GetUser.errorCode
+      description: HTTP error response code
+      type: number
+    - contextPath: GetUser.errorMessage
+      description: Reason why the API is failed
+      type: string
+    - contextPath: GetUser.id
+      description: Value of id passed as argument
+      type: string
+    - contextPath: GetUser.instanceName
+      description: Name of the instance used for testing
+      type: string
+    - contextPath: GetUser.success
+      description: Status of the result. Can be true or false.
+      type: boolean
+    - contextPath: GetUser.userName
+      description: Value of username passed as argument
+      type: string
+  - arguments:
+    - description: SCIM content in JSON format
+      name: scim
+      required: true
+    - description: An optional custom mapping that takes custom values in the SCIM
+        data into the integration.
+      name: customMapping
+    description: Creates a user
+    name: create-user
+    outputs:
+    - contextPath: CreateUser
+      description: Command context path
+      type: Unknown
+    - contextPath: CreateUser.active
+      description: Gives the active status of user. Can be true of false.
+      type: boolean
+    - contextPath: CreateUser.brand
+      description: Name of the Integration
+      type: string
+    - contextPath: CreateUser.details
+      description: Gives the raw response from API
+      type: string
+    - contextPath: CreateUser.email
+      description: Value of email ID passed as argument
+      type: string
+    - contextPath: CreateUser.errorCode
+      description: HTTP error response code
+      type: number
+    - contextPath: CreateUser.errorMessage
+      description: Reason why the API is failed
+      type: string
+    - contextPath: CreateUser.instanceName
+      description: Name of the instance used for testing
+      type: string
+    - contextPath: CreateUser.id
+      description: Value of id passed as argument
+      type: string
+    - contextPath: CreateUser.success
+      description: Status of the result. Can be true or false.
+      type: boolean
+    - contextPath: CreateUser.username
+      description: Value of username passed as argument
+      type: string
+  - arguments:
+    - description: Old SCIM content in JSON format
+      name: oldScim
+      required: true
+    - description: New SCIM content in JSON format
+      name: newScim
```

Comment:
```suggestion
      name: new_scim
```

Reply:
this is a part of design and consistent in all the integrations for User Provisioning